app/models/user.rb

```diff
@@ -1,30 +1,20 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  acts_as_authentic do |c|
+    c.login_field = :email
+    c.validate_email_field = true
+    c.validate_login_field = false
+    c.ignore_blank_passwords  = true
+    c.validate_password_field = false
+  end
+
   APPROVAL_STATUSES = %w[pending_email_verification
                          pending_approval approved
                          not_approved].freeze
-  PASSWORD_FORMAT = /\A
-    (?=.{8,}\z)        # Must contain 8 or more characters
-    (?=.*\d)           # Must contain a digit
-    (?=.*[a-zA-Z])     # Must contain a letter
-    (?=.*[[:^alnum:]]) # Must contain a symbol
-  /x.freeze
 
   validates :email, presence: true
-  validates :contact_name, presence: true
```

Comment:
Yes this feature is just focused on authenticating the user and then saving them. New user will be saved and they can't do anything till they update their info. Which then covers user has to complete registration on search.gov. I will add the validation back when I work on the edit form. But have not worked on that part yet. It will have similar flow to USAjobs. If you have any questions slack me and we can discuss.

Reply:
👍